### PR TITLE
Add Natural Earth administrative subdivisions layer

### LINKED ISSUE #170: Add worldwide ISO-3166-2 layer
Add a 2nd level admin layer. 

Basically, a collated layer of all administrative boundaries (states, provinces, ...) into a single layer

This would be in the order of about 4-5k polygons.

This would be particularly useful for the Solutions use-cases, since the geoip-processor adds 2nd level region codes as a field and Beats can be setup to produce this.

cc @nickpeihl @jsanz

### PR DESCRIPTION
**Pull request by nickpeihl**
Fixes #170 

This dataset includes a worldwide collection of administrative subdivisions produced by Natural Earth. The data is packaged as TopoJSON which is supported by Kibana 6.2+.

Some caveats here:
 - In some countries the ISO 3166-2 standard includes hierarchical levels of subdivisions. This Natural Earth dataset prioritizes topology rather than hierarchy. So the layer does not have 1:1 parity with the ISO 3166-2 standard.

- The Natural Earth dataset offers a single de facto worldview for disputed boundaries which may not be suitable for users in some countries.

- This dataset is not based on OpenStreetMap data so it may differ from existing subdivision datasets published in EMS.

- Some countries in the Natural Earth dataset only have the first level subdivisions (Bangladesh, Czechia, Fiji, Lithuania, Morocco).

### CODE REVIEW

**Comment by alexfrancoeur** [comment]
❤️

File: sources/world/world_countries.hjson [hunk 1]
```diff
@@ -200,7 +200,7 @@
     default: true
   }]
   ttl: 108000
-  weight: 1
+  weight: 2
```
[the remark this answers is not in the document]
Comment (nickpeihl) [reply sources/world/world_countries.hjson:203]: Ah, this puts higher weights at the top of the manifest. So World Countries will show up at the top and Administrative Subdivisions will show up second.

File: sources/world/administrative-subdivisions.hjson [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        {
+            type: property
+            name: label_en
+            desc: Subdivision name (English)
+        }
+    ]
+    name: Administrative subdivisions
+    legacyIds: []
+    humanReadableName: {
+      en: Administrative subdivisions
```
[the remark this answers is not in the document]
Comment (nickpeihl) [reply sources/world/administrative-subdivisions.hjson:41]: I like `World Countries Subdivisions` better. thanks.

File: sources/world/administrative-subdivisions.hjson [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      de: Verwaltungseinheit
+      zh: 行政领土实体
```
[the remark this answers is not in the document]
Comment (nickpeihl) [reply sources/world/administrative-subdivisions.hjson]: I'm getting the feeling that localization will not be very important moving forward, but 👍 for keeping the precedent.

**Comment by nickpeihl** [comment]
> I was also wondering if there are any other fields we may want to include from the NE dataset.

I think we have the fields we need for now. It's easy to add more later if want to.

File: sources/world/world_countries.hjson (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsanz) [reply sources/world/world_countries.hjson:203]: Makes sense, thanks!

**Pull request merged**